### PR TITLE
Convert pre-fork CHANGES to RST

### PR DESCRIPTION
**Pull request by BaseMax**
Hi everyone, I am new in contributing to Pillow. happy to help. ^_^

### CODE REVIEW

**Comment by BaseMax** [comment]
I used a python package for checking the errors and right now there are no errors.

```sh
$ python -m readme_renderer test.rst -o test.html
```

File: CHANGES.rst [hunk 1]
```diff
@@ -3149,7 +3147,7 @@ Changelog (Pillow)
 - Change function declaration to match Tcl_CmdProc type #1966
   [homm]
 
-- Integer overflow checks on all calls to *alloc #1781
```
Comment (wiredfool) [inline CHANGES.rst:3152]: Note that this change changes the meaning of the line.

Comment (BaseMax) [reply CHANGES.rst:3152]: Thanks for the review, Do you mean we need to show * before alloc? That was an error in RST. @wiredfool 
So `*alloc`?

File: CHANGES.rst [hunk 2]
```diff
@@ -4822,1496 +4820,1367 @@ Pre-fork
...
+- Fixed "RuntimeError: encoder error -8 in tostring" problem for
+    wide "RGB", "I", and "F" images.
 
-    + Fixed byte order issue in Image.paste(ink) (from Ka-Ping Yee).
+- Fixed line width calculation.
 
-    + Fixed off-by-0.5 errors in the ANTIALIAS code (based on input
-      from Douglas Bagnall).
+1.1.6a0
+----------------
```
Comment (radarhere) [inline CHANGES.rst:5155]:
```suggestion
-------
```
So that it lines up with "1.1.6a0"

File: CHANGES.rst [hunk 3]
```diff
@@ -4822,1496 +4820,1367 @@ Pre-fork
...
+- Fixed ImageOps.equalize when used with a small number of distinct
+    values (reported by David Kirtley).
 
-    (1.1.5c2 and 1.1.5 final released)
+- Fixed potential integer division in PSDraw.image (from Eric Etheridge).
 
-    + Added experimental PERSPECTIVE transform method (from Jeff Breiden-
-      bach).
+1.1.5c2 and 1.1.5 final
+--------------------------------
```
Comment (radarhere) [inline CHANGES.rst:5174]:
```suggestion
-----------------------
```

File: CHANGES.rst [hunk 4]
```diff
@@ -4822,1496 +4820,1367 @@ Pre-fork
...
 
-    (1.1.5c1 released)
+- Added experimental PERSPECTIVE transform method (from Jeff Breiden-
+    bach).
 
-    + Make sure "thumbnail" never generates zero-wide or zero-high images
-      (reported by Gene Skonicki)
+1.1.5c1
+----------------
```
Comment (radarhere) [inline CHANGES.rst:5180]:
```suggestion
-------
```

File: CHANGES.rst [hunk 5]
```diff
@@ -4822,1496 +4820,1367 @@ Pre-fork
...
+- Changed default "convert" palette to avoid "rounding errors" when
+    round-tripping white source pixels (reported by Henryk Gerlach and
+    Jeff Epler).
 
-    + Don't crash in "quantize" method if the number of colors requested
-      is larger than 256.  This release raises a ValueError exception;
-      future versions may return a mode "RGB" image instead (reported
-      by Richard Oudkerk).
+1.1.5b3
+----------------
```
Comment (radarhere) [inline CHANGES.rst:5193]:
```suggestion
-------
```

File: CHANGES.rst [hunk 6]
```diff
@@ -4822,1496 +4820,1367 @@ Pre-fork
...
 
-    (1.1.5b2 released)
+- Added WBMP read/write support (based on code by Duncan Booth).
 
-    + Added DPI read/write support to the PNG codec.  The decoder sets
-      the info["dpi"] attribute for PNG files with appropriate resolution
-      settings.  The encoder uses the "dpi" option (based on code by Niki
-      Spahiev).
+1.1.5b2
+----------------
```
Comment (radarhere) [inline CHANGES.rst:5203]:
```suggestion
-------
```

File: CHANGES.rst [hunk 7]
```diff
@@ -4822,1496 +4820,1367 @@ Pre-fork
...
 
-    (1.1.5b1 released)
+- Added support for Spider images (from William Baxter).  See the
+    comments in PIL/SpiderImagePlugin.py for more information on this
+    format.
 
-    + Added new Sane release (from Ralph Heinkel).  See the Sane/README
-      and Sane/CHANGES files for more information.
+1.1.5b1
+----------------
```
Comment (radarhere) [inline CHANGES.rst:5224]:
```suggestion
-------
```

File: CHANGES.rst (hunk 1 above)
Comment (radarhere) [reply CHANGES.rst:3152]: Yes, `*alloc`, thanks

**Comment by BaseMax** [comment]
Thanks for the review. @wiredfool 
Your review merged and the next one was fixed.

File: CHANGES.rst [hunk 8]
```diff
@@ -1527,7 +1527,6 @@ Changelog (Pillow)
 
 - Deprecate PIL.*ImagePlugin.__version__ attributes #3628
   [jdufresne]
```
Comment (radarhere) [inline CHANGES.rst:1529]:
```suggestion
  [jdufresne]

```
Why remove this blank line?

File: CHANGES.rst [hunk 9]
```diff
@@ -3139,7 +3138,6 @@ Changelog (Pillow)
 
 - Changed depends/install_*.sh urls to point to github pillow-depends repo #1983
   [wiredfool]
```
Comment (radarhere) [inline CHANGES.rst:3140]:
```suggestion
  [wiredfool]

```

**Comment by BaseMax** [comment]
Thanks for your review. These merged. @radarhere

File: CHANGES.rst [hunk 10]
```diff
@@ -3149,7 +3149,7 @@ Changelog (Pillow)
 - Change function declaration to match Tcl_CmdProc type #1966
   [homm]
 
-- Integer overflow checks on all calls to *alloc #1781
+- Integer overflow checks on all calls to ``*alloc`` #1781
```
Comment (radarhere) [inline CHANGES.rst:3152]:
```suggestion
- Integer overflow checks on all calls to \*alloc #1781
```
Experimenting, I find that this also works.

**Comment by radarhere** [comment]
I've created https://github.com/MaxFork/Pillow/pull/2 with some more suggestions.

**Review by radarhere — APPROVED** [review]

**Comment by BaseMax** [comment]
Happy contributing, hope to do another thing soon after this one. :heart:

**Pull request merged**